telco-hub/configuration/reference-crs/required/acm/pre-requeriments.md

```diff
@@ -0,0 +1,22 @@
+Some previous configured components:
+   * ACM requires storage configured in order to use the AgentServiceConfig
+   * MCO requires a pull-secret imported on the NS `open-cluster-management-observability` as a Secret.
+   * MCO requires an S3 compatible bucket storage with the connectivity info as:
+
+```yaml
```

Comment:
We should provide this, with "example" values as a CR as well as the documentation here..

Reply:
@imiller0, I agree. But, where to document? I created this short document to remind some minor things. Is oka if I add here examples?

Reply:
Yes, I think it is ok (and good) to have example CRs included that the user will need to update with the correct values.